src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -335,6 +278,52 @@ public void OnHide()
         Hide();
     }
 
+    /// <summary>
+    ///   Searches the processes description string for compound name and input action
+    ///   "keys" and turns them into a BBCode string with matching values.
+    /// </summary>
+    private string ParseProcessesDescription()
+    {
+        if (string.IsNullOrEmpty(ProcessesDescription))
+            return string.Empty;
+
+        var inputEvents = Settings.GetCurrentlyAppliedControls();
+
+        var result = Regex.Replace(TranslationServer.Translate(ProcessesDescription), @"\[(.*?)\]", found =>
+        {
+            var parsed = string.Empty;
+
+            var value = found.Groups[1].Value;
+
+            // Parse compound names
+            if (SimulationParameters.Instance.DoesCompoundExist(value))
+            {
+                var compound = SimulationParameters.Instance.GetCompound(value);
+
+                parsed = $"[b]{compound.Name}[/b] [font=res://src/gui_common/fonts/" +
+                    $"BBCode-Image-VerticalCenterAlign-3.tres] [img=20]{compound.IconPath}[/img][/font]";
+            }
+
+            // Parse input actions
+            if (InputMap.HasAction(value))
+            {
+                var events = inputEvents.Data[value];
+
+                for (int i = 0; i < events.Count; i++)
+                {
+                    parsed += $"[b][{KeyNames.Translate(events[i].Code)}][/b]";
```

Comment:
I think this could share some logic with the key button prompts, as these really should share the same logic to determine what to show in the GUI text and on the button press prompts.